quickjs.c

```diff
@@ -33742,6 +33742,30 @@ static JSValue js_global_isFinite(JSContext *ctx, JSValueConst this_val,
     return JS_NewBool(ctx, res);
 }
 
+static JSValue js_microtask_job(JSContext *ctx,
+                                int argc, JSValueConst *argv)
+{
+    JSValueConst func = argv[0];
+    JSValue ret = JS_Call(ctx, func, ctx->global_obj, 0, NULL);
+    if (JS_IsException(ret))
+        return ret;
+    JS_FreeValue(ctx, ret);
+    return JS_UNDEFINED;
```

Comment:
Can be simplified to:
```suggestion
    return JS_Call(ctx, argv[0], ctx->global_obj, 0, NULL);
```
JS_ExecutePendingJob() already takes care of freeing the return value and checking whether it's an exception.

Reply:
👍